server/src/testFixtures/java/org/elasticsearch/test/IntegTestCase.java

```diff
@@ -272,6 +274,7 @@
 @UseJdbc
 @UseHashJoins
 @UseRandomizedSchema
+@UseRandomizedOptimizerRules(value = 1, disablePercentage = 1, alwaysKeep = {RemoveRedundantFetchOrEval.class, MergeFilterAndCollect.class})
```

Comment:
`RemoveRedundantFetchOrEval` and `MergeFilterAndCollect` have to be always activated otherwise a lot of tests will fail.